modules/apps/commerce/commerce-warehouse-web/src/main/java/com/liferay/commerce/warehouse/web/internal/display/context/CommerceInventoryWarehousesDisplayContext.java

```diff
@@ -14,39 +14,35 @@
 
 package com.liferay.commerce.warehouse.web.internal.display.context;
 
-import com.liferay.commerce.country.CommerceCountryManager;
 import com.liferay.commerce.frontend.model.HeaderActionModel;
 import com.liferay.commerce.inventory.model.CommerceInventoryWarehouse;
 import com.liferay.commerce.inventory.service.CommerceInventoryWarehouseService;
-import com.liferay.commerce.product.display.context.helper.CPRequestHelper;
+import com.liferay.commerce.product.display.context.util.CPRequestHelper;
```

Comment:
@christian-chiappa @andrea-ale-sbarra why has CPRequestHelper been moved to util package?